src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -274,7 +274,14 @@ valid_file_mem_backends = @DEFVALIDFILEMEMBACKENDS@
 # The value means the number of pcie_root_port
 # This value is valid when hotplug_vfio_on_root_bus is true and machine_type is "q35"
 # Default 0
-#pcie_root_port = 2
+pcie_root_port = 2
+
+# Delay in seconds to attach the large PCI Bar devices
+# Use this parameter when using some large PCI bar devices, and the guest kernel is 5.x
+# The value means the delay in seconds for attach a large PCI device
```

Comment:
Isn't this line duplicated with line 279?

Reply:
L279 is the title of this parameter.  
L281 is the description of the value.

Reply:
If the description doesn't add useful information to the title, there's not much point to it.

In this case being specific about where this delay time starts would be useful.

Reply:
Updated description @liubin  @dgibson